rolemanager/lib/Utils/Internacionalization/internacionalization.dart

```diff
@@ -6,7 +6,12 @@ enum ENUM_INTER {
   buttonloginInvitation,
   lbbuttonlogin,
   lbmailinpult,
-  lbpassinpult
+  lbpassinpult,
+  lbmodalivite,
```

Comment:
ENG:
Change variable name to "**lbmodalinvite**"
PT:
Mudar o nome da variavel para "**lbmodalinvite**"